the-collab-lab/tcl-46-smart-shopping-list
# Added TS to ESLint Config

## Description

Adjusted config so that ESLint will not throw errors when writing `ts` and `tsx` files.

## Related Issue

Closes #38 


## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
|    | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|  ✓   | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates

### Before

![image](https://user-images.githubusercontent.com/77255525/185219902-93b7cdb5-6f3e-4e67-b3e9-8368088429fa.png)
![image](https://user-images.githubusercontent.com/77255525/185219976-5647e024-1f09-45bd-8a4a-15a5ac6c4e46.png)


### After

![image](https://user-images.githubusercontent.com/77255525/185220028-488c1544-fa72-4def-9afe-1b8c0f87c815.png)

## Testing Steps / QA Criteria

Switch any `js` or `jsx` file to a `ts` or `tsx` file to see that the Intellisense does not protest.
